Sources/Sockets/TCP/TCPInternetSocket.swift

```diff
@@ -0,0 +1,88 @@
+import libc
+
+public typealias TCPDuplexProgram = TCPServer & TCPClient
```

Comment:
Should this just be `TCPReadWriteProgram`? The duplex naming convention throughout makes me think of a house, not sure if its a convention